CMakeLists.txt

```diff
@@ -62,12 +62,8 @@ endif()
 # Enable ASAN if possible and requested
 if(TB_ENABLE_ASAN)
     message(STATUS "Enabling ASan")
-    if(COMPILER_IS_CLANG)
-        add_compile_options(-fsanitize=address)
-        add_link_options(-fsanitize=address)
-    else()
-        message(WARNING "TB isn't set up to enable ASan for compiler ${CMAKE_CXX_COMPILER_ID}")
-    endif()
+    add_compile_options(-fsanitize=address)
```

Comment:
What about MSVC?

Reply:
It'll need something else I think.
https://devblogs.microsoft.com/cppblog/addresssanitizer-asan-for-windows-with-msvc/

I can restore the warning if it's not GCC or Clang and you try to use TB_ENABLE_ASAN